tornado/httputil.py

```diff
@@ -520,7 +520,7 @@ class HTTPMessageDelegate(object):
     .. versionadded:: 4.0
     """
 
-    # TODO: genericize this class to avoid exposing the Union.
```

Comment:
This one was intentional - "genericize" in the sense of parameterized types, not generalization.